transform/operations.go

```diff
@@ -357,7 +360,38 @@ func (c *convertToFloat64) transform(raw interface{}) (interface{}, error) {
 	}
 }
 
-// requiredArgs checks the given args map to make sure it contains the required args and only the required args.
+// convertToInt64 is a transformOperation which converts various types to int64.
+type convertToInt64 struct {
+	args map[string]string
+}
+
+func (c *convertToInt64) init(args map[string]string) error {
+	return nil
+}
+
+func (c *convertToInt64) transform(raw interface{}) (interface{}, error) {
+	switch in := raw.(type) {
+	case string:
+		return strconv.ParseInt(in, 2, 64)
+	case float64:
```

Comment:
There is also `float32`, if that matters.

Reply:
https://github.com/GannettDigital/jstransform/pull/123/commits/3f653fe43b2c06e825cfab875c4033406ebf0ee8